trin-core/src/portalnet/types.rs

```diff
@@ -345,6 +345,137 @@ impl FromStr for HexData {
     }
 }
 
+pub enum ContentKey {
+    StateContentKey(StateContentKey),
+    HistoryContentKey(HistoryContentKey),
+}
+
+pub enum StateContentKey {
+  AccountTrieNode(AccountTrieNode),
```

Comment:
I worry that this duplicated name can be a bit confusing. Maybe something like this?
```suggestion
  AccountTrieNodeKey(AccountTrieNode),
```

Reply:
idk why I added key to everything else except for state keys, probably cuz i thought the names were getting too long lol. will add